Backend/src/main/java/ca/mcgill/cooperator/model/CoopStatus.java

```diff
@@ -1,6 +1,7 @@
 package ca.mcgill.cooperator.model;
 
 public enum CoopStatus {
+    FINISHED,
     COMPLETED,
```

Comment:
what the difference between completed and finished

Reply:
Read the PR description

Reply:
jk i see now but hopefully this isnt confusing if u end up in finished state and dot realize it doesnt mean ur actually done

Reply:
maybe we should make a page explaining what you have to be doing during each of the states

Reply:
Yeah fair, we can make it more obvious in the frontend by letting the student know their Coop has to be finalized by an Admin before it's officially "done"